UM/Settings/Interfaces.py

```diff
@@ -199,4 +205,10 @@ def getEmptyInstanceContainer(self) -> "InstanceContainer":
         raise NotImplementedError()
 
     def isReadOnly(self, container_id: str) -> bool:
-        raise NotImplementedError()
\ No newline at end of file
+        raise NotImplementedError()
+
+    def setReadOnlyExplicitly(self, container_id: str) -> None:
```

Comment:
I'd renamed this to `setExplicitReadOnly()` so it's consistent with `isExplicitReadOnly`

Reply:
Basically the same as the first comment, so also done :-)